fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/storage/TransferVariant.java

```diff
@@ -46,27 +43,24 @@ public interface TransferVariant<O> {
 	O getObject();
 
 	/**
-	 * Return the underlying tag.
-	 *
-	 * <p><b>NEVER MUTATE THIS NBT TAG</b>, if you need to mutate it you can use {@link #copyNbt()} to retrieve a copy instead.
+	 * @return The {@link ComponentChanges} of this variant.
```

Comment:
We should keep a method to copy the components, and a method to directly get them.

Reply:
As above, they are immutable.